.stylelintrc

```diff
@@ -0,0 +1,71 @@
+{
+  "extends": "stylelint-config-recommended-scss",
+  "plugins": [
+    "stylelint-order",
+    "stylelint-scss"
+  ],
+  "rules": {
+    "at-rule-empty-line-before": [
+      "always", {
+        "ignoreAtRules": [ "after-comment", "else" ]
+      }
+    ],
+    "block-opening-brace-space-before": "always",
+    "block-closing-brace-newline-after": [
+      "always", {
+        "ignoreAtRules": [ "if", "else" ]
+      }
+    ],
+    "at-rule-name-space-after": [
+      "always", {
+        "ignoreAtRules": [ "after-comment" ]
+      }
+    ],
+    "color-hex-case": "lower",
+    "color-hex-length": "short",
+    "color-no-invalid-hex": true,
+    "declaration-colon-space-after": "always",
+    "declaration-colon-space-before": "never",
+    "indentation": "tab",
+    "no-descending-specificity": null,
+    "no-eol-whitespace": true,
+    "property-no-vendor-prefix": true,
+    "rule-empty-line-before": "always",
+    "selector-type-no-unknown": [
+      true, {
+        "ignoreTypes": "as-link",
```

Comment:
I thought this was some special SASS thing I didn't know about while I was working on this, but looking at it now I realized it must've been a typo for `.as-link`.